readme.rst

```diff
@@ -229,6 +229,32 @@ your doc files were placed (repository root or the ``docs`` directory):
    make install
    make html
 
+Build a PDF
+^^^^^^^^^^^
+
+The documentation can build a PDF locally with the following command:
+
+.. code-block:: none
+
+    make pdf
+
+PDF generation requires some system files. If these files are not found, a prompt will be presented and the generation will stop.
+
+Required packages can be installed with:
```

Comment:
On Linux, ...

Reply:
I assume that this is the case for all of the starter pack functions. For example, `woke` only functions on Linux, and specifically Ubuntu or Ubuntu derivatives - as it installs a snap which isn't available by default for a number of other Linux distros.

Reply:
```suggestion
On Linux, required packages can be installed with:
```